### ISSUE TITLE
Fatal error when requiring RBLTrackerException.php

### ISSUE THREAD

**Issue by YasinUzn** [rbltracker/rbltracker-php#1]
I encountered the following error while using the RBLTracker package on Laravel 7.4 project:

This error code comes from Laravel:
```
<warning>PHP Warning: require_once(/our_app/vendor/rbltracker/sdk/RBLTracker/RBLTracker/Exceptions/RBLTrackerException.php): failed to open stream: No such file or directory in /our_app//vendor/rbltracker/sdk/RBLTracker/Client.php on line 69</warning>
PHP Fatal error: require_once(): Failed opening required '/our_app/vendor/rbltracker/sdk/RBLTracker/RBLTracker/Exceptions/RBLTrackerException.php' (include_path='.:/opt/rh/rh-php73/root/usr/share/pear:/opt/rh/rh-php73/root/usr/share/php') in /our_app/vendor/rbltracker/sdk/RBLTracker/Client.php on line 69
```

To simplify the process and see if the exception class can be autoloaded from a standalone PHP script wrote a simple test.php:

```
<?php

require __DIR__.'/vendor/autoload.php';

var_dump(class_exists(\RBLTracker\Client::class));
```

This returns true

If I try 
 
```
<?php

require __DIR__.'/vendor/autoload.php';

var_dump(class_exists(\RBLTracker\Client\RBLTrackerException::class));

returns false
```

Any suggestions?

**Comment by mikepultz** [comment]
Hey @YasinUzn,

The exception class is \RBLTracker\Exceptions\RBLTrackerException not \RBLTracker\Client\RBLTrackerException

Try with that path and it should fix your issue.

Mike

**Comment by YasinUzn** [comment]
@mikepultz thanks for the quick reply sorry I pasted wrong here mb. The code was like that already:

This is directly from test.php no relation with Laravel. 

test.php looks like this:

```
<?php

require __DIR__.'/vendor/autoload.php';

var_dump(class_exists(\RBLTracker\Client::class));

var_dump(class_exists(\RBLTracker\Exceptions\RBLTrackerException::class));
```

Here is the response I got when I trigger that php file:

```
$ php test.php
```
This is the output:
```
bool(true)
<?

//
// This file is part of the RBLTracker PHP Wrapper package.
//
// (c) Mike Pultz <mike@mikepultz.com>
//
// For the full copyright and license information, please view the LICENSE
// file that was distributed with this source code.
//

namespace RBLTracker\Exceptions;

class RBLTrackerException extends \Exception {}
PHP Warning:  require_once(RBLTracker/Exceptions/RBLTrackerException.php): failed to open stream: No such file or directory in /our_app/vendor/rbltracker/sdk/RBLTracker/Client.php on line 72
PHP Fatal error:  require_once(): Failed opening required 'RBLTracker/Exceptions/RBLTrackerException.php' (include_path='.:/opt/rh/rh-php73/root/usr/share/pear:/opt/rh/rh-php73/root/usr/share/php') in /our_app/vendor/rbltracker/sdk/RBLTracker/Client.php on line 72
```

This is exact same error I also got from Laravel when the response fails too FYI.

When test.php looks like this:

```
<?php

require __DIR__.'/vendor/autoload.php';

var_dump(class_exists(\RBLTracker\Exceptions\RBLTrackerException::class));
```

Here is the output:

```
<?

//
// This file is part of the RBLTracker PHP Wrapper package.
//
// (c) Mike Pultz <mike@mikepultz.com>
//
// For the full copyright and license information, please view the LICENSE
// file that was distributed with this source code.
//

namespace RBLTracker\Exceptions;

class RBLTrackerException extends \Exception {}
bool(false)
```

Hope this is clear. Any suggestions?

**Comment by mikepultz** [comment]
Hey @YasinUzn,

Yes- I see- I appears I used "short open tags" (just <? not <?php)  in the RBLTracker/Exceptions/RBLTrackerException.php file; I'm not sure why, I don't use it in any of the other files. I didn't notice when I tested because i have short open tags enabled on my dev systems.

I'll correct this in the next release; but the easiest fix, is to just change the first line in RBLTracker/Exceptions/RBLTrackerException.php to `<?php` instead of just `<?` and it should fix your issue.

Mike
